hack/docs/check.sh

```diff
@@ -14,6 +14,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+source "$(dirname "$0")/../../test/common.sh"
```

Comment:
Why do we need to add this one here?

Reply:
Hi @camilamacedo86 

It seems we need to prepare a kb for the pipeline based on the above discussion: https://github.com/kubernetes-sigs/kubebuilder/pull/3231#issuecomment-1465510380